packages/terra-consumer-nav/src/components/nav-help/NavHelp.scss

```diff
@@ -48,6 +47,12 @@
     }
   }
 
+  .border-top-except-first-element {
```

Comment:
Do not name classes after what they are styling, name them after what they are doing.  Describe the container element this is applied to and style that.

Reply:
Ah I see this is reused across components... don't use CSS classes like that. Put this style onto each component that needs it.